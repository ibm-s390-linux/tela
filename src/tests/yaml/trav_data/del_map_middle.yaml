map1:
  child1:
delete:
  child2:
map3:
  child3:
