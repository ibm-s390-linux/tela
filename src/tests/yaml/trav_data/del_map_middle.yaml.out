Before (root=non-null):
==================================
  map1:
    child1:
  delete:
    child2:
  map3:
    child3:
==================================

Callback:
==================================
map1                                    : map1:
map1/child1                             :   child1:
delete                                  : delete:
*** Deleting node
map3                                    : map3:
map3/child3                             :   child3:
==================================

After (root=non-null):
==================================
  map1:
    child1:
  map3:
    child3:
==================================
